demosplan/DemosPlanCoreBundle/Command/Data/GenerateStatementFragmentCommand.php

```diff
@@ -96,7 +96,7 @@ protected function configure(): void
     protected function initialize(InputInterface $input, OutputInterface $output): void
     {
         $functionalUser = new FunctionalUser();
-        $functionalUser->setDplanroles([Role::PLANNING_AGENCY_ADMIN]);
+        $functionalUser->addRoles([Role::PLANNING_AGENCY_ADMIN]);
```

Comment:
set and add is different. Why did you change this? With set you replace roles, with add you add them to existing ones